src/braket/pennylane_plugin/translation.py

```diff
@@ -57,19 +40,181 @@ def supported_operations() -> FrozenSet[str]:
     Returns:
         FrozenSet[str]: The names of the supported operations
     """
-    return frozenset(_OPERATION_MAP.keys())
+    return frozenset(op.__name__ for op in _translate_operation.registry) - {"object"}
```

Comment:
I LOVE the use of `frozetset`; immutability is the best thing since sliced bread in programming